.github/workflows/typescript-publish.yml

```
@@ -0,0 +1,133 @@
+            if [[ ${GITHUB_REF##*/} == "7."**".x" ]]; then
+              echo "[INFO] using ${GITHUB_REF##*/} tag"
+              DIST_TAG="next-${GITHUB_REF##*/}"
+            else
+              echo "[INFO] using "next" tag"
+              DIST_TAG=next
+            fi
+          fi
+          echo "npm_dist_tag=$DIST_TAG" >> $GITHUB_OUTPUT
+      - name: Check existing release tags
+        if: github.event_name == 'workflow_dispatch'
+        run: |
```

actionlint (1.7.12)

[error] property "forcerecreatetags" is not defined in object type {release-remake: string; release-version: string}

(expression)